pandas/core/arrays/integer.py

```diff
@@ -78,7 +78,8 @@ def itemsize(self):
 
     @classmethod
     def construct_array_type(cls):
-        """Return the array type associated with this dtype
+        """
```

Comment:
Is this docstring not inherited from the base class?

Reply:
not as far as I'm aware. Should it be?

Reply:
It looks like `inspect.getdoc(object)` https://docs.python.org/3/library/inspect.html?highlight=inspect#inspect.getdoc could get the baseclass docstring.

I'm using VSCode and not showing baseclass docstring when no docstring exists on derived class.

`_IntegerDtype().construct_array_type.__doc__` returns None if no docstring exists on derived class.

Reply:
ok this is odd then I agree with @TomAugspurger that these should inherit the baseclass, odd that they are not.